pysingfel/solvent_form_factor/saxs_profile.py

```diff
@@ -282,17 +294,13 @@ def calculate_profile_partial (prof,particles,saxs_sa,ft,vff,dff,verbose,c1,c2,f
     :param ft: form factor table object
     :param vff: table of vacuum form factors
     :param dff:  table of dummy form factors
-    :param verbose: writing to files for testing
     :param c1: excluded volume parameter
     :param c2: hydration layer parameter
     :return intensity: the intensity
     """ 
     
     # can write output files containing atomtypes, vacuum/dummy form factors
-    if verbose ==1:
```

Comment:
Are we not testing these anymore? Or are these produced on the fly during testing somewhere else?

Reply:
We are testing our python code on the fly vs. libsaxs files in the test cases. But in the code itself we are no longer writing of most of these files.